modules/mapbox/src/deck-utils.ts

```diff
@@ -9,7 +9,7 @@ type UserData = {
   isExternal: boolean;
   currentViewport?: WebMercatorViewport | null;
   mapboxLayers: Set<MapboxLayer<any>>;
-  mapboxVersion: {minor: number; major: number};
+  // mapboxVersion: {minor: number; major: number};
```

Comment:
Version parsing is no longer used, keeping the code in case another breaking change is introduced in the base map libraries in the future.